cmd/common/transaction.go

```diff
@@ -338,10 +354,33 @@ func PrintTransactionBeforeSigning(npa *NPASelection, tx interface{}) {
 
 // PrintSignedTransaction prints a signed transaction.
 func PrintSignedTransaction(sigTx interface{}) {
-	// TODO: Add some options for controlling output.
```

Comment:
`Print*Transaction` method names seem outdated. So `PrintSignedTransaction()` is now used to print unsigned transactions as well? I would refactor the code a bit so `BroadcastTransaction` is not called in offline and txUnsigned mode anymore (do we want to broadcast unsigned tx in any scenario?)

Reply:
Renamed `PrintSignedTransaction` to `ExportTransaction` as that seems more appropriate now.

Also there are now separate `BroadcastTransaction` and `BroadcastOrExportTransaction` and usage has been updated.